website/project/model.py

```diff
@@ -2300,6 +2303,13 @@ def update_node_wiki(self, name, content, auth):
         )
         new_page.save()
 
+        wiki_changed.send(
+            name,
```

Comment:
> Connected functions are always passed the object that caused the signal to be emitted.

https://pythonhosted.org/blinker/

That means you should pass the node (`self`) as the first argument to send, and adjust the subscriber functions accordingly.